modular_skyrat/modules/reagent_forging/code/forge.dm

```diff
@@ -178,14 +184,14 @@
 		to_chat(user, span_notice("[src] requires you to be more experienced!"))
 		return
 	if(user_smithing_skill >= SKILL_LEVEL_APPRENTICE)
-		goliath_ore_improvement = 3
+		goliath_ore_improvement = MAX_UPGRADE_GOLIATH
 		forge_level = FORGE_LEVEL_ONE
 	if(user_smithing_skill >= SKILL_LEVEL_EXPERT)
-		sinew_lower_chance = 100
-		current_sinew = 10
+		sinew_lower_chance = MAX_UPGRADE_SINEW * 10 //100, just written funny!
+		current_sinew = MAX_UPGRADE_SINEW
 		forge_level = FORGE_LEVEL_TWO
 	if(user_smithing_skill >= SKILL_LEVEL_MASTER)
-		current_core = 3
+		current_core = MAX_UPGRADE_REGEN
 		forge_level = FORGE_LEVEL_THREE
 		create_reagent_forge()
```

Comment:
I feel like this would make more sense as a switch statement. Or at the very least, a switch statement for the variable assignation for `forge_level` because otherwise you constantly set it to 1, then to 2, then to 3, if you're a MASTER.

Reply:
it would have a lot of repeat code if it was a switch statement, since it is intended to go through all of the ifs. Also, you can only upgrade it once up to its level, so a level 2 cant be downgraded to level 1, etc.